src/braille.jl

```diff
@@ -8,37 +11,42 @@ Binary dither with algorithm `alg`, then print image in braille.
 
 ## Keyword arguments
   - `invert`: invert Unicode output, defaults to `false`
+  - `to_string`: return a string instead of printing to terminal. Defaults to `false`.
 
 All keyword arguments for binary dithering methods can be used.
 """
-function braille(img::GenericImage, alg::AbstractDither; invert=false, kwargs...)
+function braille(
+    img::GenericImage,
+    alg::AbstractDither;
+    invert::Bool=false,
+    to_string::Bool=false,
+    kwargs...,
+)
     d = dither(Bool, Gray.(img), alg; kwargs...)
```

Comment:
If `img` is already `AbstractArray{<:Gray}`, this will be like an unnecessary `collect(img)`.

Reply:
[I added dispatches for gray and binary images.](https://github.com/JuliaImages/DitherPunk.jl/pull/91/commits/c319d1b0b4be0d74507b0158aad8c231dcddd319)